yarn.lock

```diff
@@ -0,0 +1,2335 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+abbrev@1:
```

Comment:
A lot of new dependencies have been introduced - are you concerned with the project becoming bloated with unnecessary dependencies?